core/src/ledger/faucet.rs

```diff
@@ -43,7 +43,12 @@ pub struct Challenge {
     /// PoW difficulty
     pub difficulty: u8,
     /// The counter value of the `transfer.target`
+    /// TODO: What does this mean though?
```

Comment:
The current counter value is read from storage for the given `transfer.target`. The VP then checks that this matches what's in pre-storage and incremented in post-storage